spec/helpers/search_helper_spec.rb

```diff
@@ -1,6 +1,68 @@
 require 'rails_helper'
 
 RSpec.describe SearchHelper, type: :helper do
+
+  describe "#paginate" do
+
+    let (:page_params) {{
+      :total_pages=>3,
+      :previous_link=>"/petitions?page=1&state=all",
+      :next_link=>"/petitions?state=all",
+      "model" => Petition
+    }}
```

Comment:
My personal preference is to use `do ... end` for multi-line `let` blocks and I think that's what we've used elsewhere in the project. When making a PR, especially to a public project, you should try to adapt your code style to that of the project's even when it isn't enforce via rubicop 😄